### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by AnEmortalKid**
adding prereq for maven / java 8 && -DskipTests hint

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,12 +22,21 @@ from [kcdc.info](http://www.kcdc.info/), and returns results about these talks.
...
+* [Maven 3.x](https://maven.apache.org/index.html)
+* [Java 8](http://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html)
+
 Assemble the service:
 
 ```bash
 mvn clean package
 ```
 
+Optionally, you can add `-DskipTests` if you notice test failures when running `mvn clean package`:
```
Comment (cchesser) [inline README.md]: I would prefer to not advertise anyone to skip tests when building locally (as this is hinting other problems will probably still occur on their setup). Shall we just include a link here for downloading the pre-built JAR for running the class (if they don't have the pre-reqs)?

Comment (AnEmortalKid) [inline README.md]: yeah, that seems like a good approach! I'll see if I can get those random thread failures fixed! (I think it comes from the wiremock rule) but I haven't looked 2 deep

**Comment by cchesser** [comment]
I will merge these notes and include a link to the artifact that can be downloaded (vs. having the skipTest).

**Comment by cchesser** [comment]
Applied these changes here: 28600053d2a9288c5834081c83f220bfdf58a03a

**Pull request closed without merging**